lib/newman_conway.rb

```diff
@@ -1,7 +1,24 @@
 
 
-# Time complexity: ?
-# Space Complexity: ?
+# Time complexity: ? o(n)
+# Space Complexity: ? o(n)
 def newman_conway(num)
-  raise NotImplementedError, "newman_conway isn't implemented"
-end
\ No newline at end of file
+  raise ArgumentError if num <= 0
+  return 1 if num == 1
```

Comment:
Because the method is supposed to return a string. 

```suggestion
  return "1" if num == 1
```